config/routes.rb

```diff
@@ -3,8 +3,17 @@
   # root "articles#index"
   # Defines the root path route ("/")
 
-  root "users#index"
+  root "users#index", as: "users_home"
   resources :users, only: [:show]
   get "users/:user_id/posts", to: "posts#index"
-  get "users/:user_id/posts/:post_id", to: "posts#show"
+  get "users/:user_id/posts/:post_id", to: "posts#show", as: "posts_show"
+
+  get "post/new", to: "posts#new", as: "new_post" # new
+  post "post/create", to: "posts#create", as: "post_create" #create
+
+  post "like/create", to: "likes#create", as: "like_create" #create
+
+  get "users/:user_id/post/:post_id/comment/new", to: "comments#new" #new
+  post "comment/create", to: "comments#create", as: "comment_create" #create
+
```

Comment:
- [OPTIONAL] Configure your routes this way would make your routes clean and easier to read.

```
   resources :users, only: [:index, :show] do
       resources :posts, only: [:index, :show]
   end
```